//exception handling
console.log(`start execution`);

exp = "10/2+5**"
try {
    result = eval(exp)
    console.log(result);
}
catch (error) {
   console.log(`expression is wrong`);
   
}
num1 =10
num2 =50
sum = num1 + num2
console.log(sum);
